ToDo/index.html

```diff
@@ -5,6 +5,13 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>ToDo</title>
+    <link rel="stylesheet" href="src/css/todo/index.css" />
   </head>
-  <body></body>
+  <body>
+    <!-- START OF CODE  -->
+    <div class="todo-container">
```

Comment:
wrapp it into div with classname todo-wrapper

Reply:
changed